biologging-sensor-data/src/components/graphs/line/LineGraph.tsx

```diff
@@ -73,19 +79,52 @@ export default function LineGraph({ events, sensor }: { events: Event[], sensor:
   }, [events]);
 
   useEffect(() => {
-    setOptions({
-      responsive: true,
-      plugins: {
-        legend: {
-          position: 'bottom' as const,
-        },
-        title: {
-          display: true,
-          text: sensor.toUpperCase(),
-        },
+    const fetchData = async () => {
+      try {
+        const response = await get<any>(`dataset/${events[0].datasetID}`);
+        const unitOfMeasure = response.unitsReported[events[0].valuesMeasured.indexOf(sensor)];
+        setOptions(prevOptions => ({
+          ...prevOptions,
+          scales: {
+            ...prevOptions.scales,
+            y: {
+              ...prevOptions.scales.y,
+              title: {
+                display: true,
+                text: `${sensor} (${unitOfMeasure})`,
+              },
+            },
+          },
+        }));
+      } catch (error) {
```

Comment:
Why are you dealing with the error handling? It should be dealt with the next sprint, so please just remove it.

Reply:
@zrohacova , I had already worked on this part before the story card moved to the next sprint. Why remove it now, especially since it doesn't create any merge conflicts and resolves the API error?

Reply:
Because it is not thought over on how to do it and we will probably have to do it differently, so it will need to be changed.